app/models/manageiq/providers/container_manager.rb

```diff
@@ -12,7 +12,7 @@ class ContainerManager < BaseManager
     has_many :container_limits, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_image_registries, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_images, :foreign_key => :ems_id, :dependent => :destroy
-    has_many :persistent_volumes, :as => :parent, :dependent => :destroy
+    has_many :persistent_volumes, :foreign_key => :parent_id, :dependent => :destroy
```

Comment:
Since we're doing something deliberately wrong (but happening to work), and on euwe only, please add explanatory comment.